blog/docs/articles/knative-backstage-plugins.md

```diff
@@ -0,0 +1,62 @@
+# Knative and Backstage
+
+**Author: Ali Ok, Principal Software Engineer @ Red Hat**
+
+## What's the vision?
+
+[Backstage](https://backstage.io/) is a platform for building developer portals. It provides a unified way to manage and visualize the different resources that developers use in their daily work. 
+
+However, Backstage is not your next generation Kubernetes dashboard. It is not designed to replace the Kubernetes dashboard or any other Kubernetes management tool.
+
+With these in mind, our vision is to provide a way to visualize and partially manage Knative resources in Backstage. However, these are to be considered as read-only resources and we aim to show
+resources that are relevant to the developer persona.
+
+## Plugins and backends
+
+The plugins are the frontend part of the Backstage plugin. They are responsible for rendering the UI and communicating with the backend. The backend is responsible for talking to the Kubernetes API server and providing the necessary information to the frontend. Currently, we only have 1 plugin and it requires a backend to work. We think that most of the plugins we might provide in the future will require a backend-for-frontend to work.
+
+## The Event Mesh plugin
+
+[The Event Mesh plugin](https://github.com/knative-extensions/backstage-plugins) is a Backstage plugin that allows you to view and manage Knative Eventing resources. The plugin talks to a special backend that runs in the Kubernetes cluster and communicates with the Kubernetes API server. 
+
+The plugin is not designed to be the ultimate tool for managing Knative Eventing resources. It is designed to be a developer-friendly tool that allows developers to visualize and partially manage the resources that are relevant to them. Given these constraints, the plugin is designed to be read-only, at least for the time being.
+
+This plugin leverages Backstage's [entity provider](https://backstage.io/docs/features/software-catalog/external-integrations/#custom-entity-providers) and [entity processor](https://backstage.io/docs/features/software-catalog/external-integrations/#custom-processors) concept. The entity provider is responsible for fetching the resources from the backend and the entity processor is responsible for processing the resources and making them available to the frontend. The provider is the part that talks to the custom backend.
+
+Plugin fetches Knative EventType, Broker, and Trigger resources from the backend. Actually, it doesn't fetch the resources as is since the Kubernetes resources are not particularly designed for using them in this kind of UI, but it fetches better suited representations of the resources where some fields are ignored, some aggregated, and some transformed.
```

Comment:
```suggestion
The plugin fetches Knative EventType, Broker, and Trigger resources from the backend. Actually, it doesn't fetch the resources as is since the Kubernetes resources are not particularly designed for using them in this kind of UI, but it fetches better suited representations of the resources where some fields are ignored, some aggregated, and some transformed.
```
nit